README.md

```diff
@@ -1,2 +1,193 @@
 # hello-react-front-end
-This is a front-end application created using redux react to fetch and display data from rails back end app using APIs.
+<a name="hello-react-front-end"></a>
+###hello-react-front-end
```

Comment:
- **[OPTIONAL]** Kindly remove this line for professional documentation 👍🏽 

![image](https://github.com/ikoote1/hello-react-front-end/assets/58642949/5057723c-4708-475a-bb74-43b9199238ac)